snap/hooks/install

```diff
@@ -1,9 +1,28 @@
 #!/bin/bash -e
 
+# setup env variables - none of these paths are setup for hooks like they are for
+# apps in snapcraft, so we have to do this ourselves
+export PATH="$SNAP/usr/sbin:$SNAP/usr/bin:$SNAP/sbin:$SNAP/bin:$PATH"
+# for LD_LIBRARY_PATH, we need to handle the different architecture paths
+case $(arch) in
+    x86_64)
+        MULTI_ARCH_PATH="x86_64-linux-gnu";;
+    arm*)
+        MULTI_ARCH_PATH="arm-linux-gnueabihf";;
+    aarch64)
+        MULTI_ARCH_PATH="aarch64-linux-gnu";;
+    *)
+        echo "architecture $ARCH not supported"
+        exit 1
+        ;;
+esac
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$SNAP/lib:$SNAP/usr/lib:$SNAP/lib/$MULTI_ARCH_PATH:$SNAP/usr/lib/$MULTI_ARCH_PATH"
+export LD_LIBRARY_PATH=$SNAP_LIBRARY_PATH:$LD_LIBRARY_PATH
```

Comment:
Where is SNAP_LIBRARY_PATH defined?

Reply:
It's exported by snapd when running a snap:
```bash
$ snap run --shell hello
To run a command as administrator (user "root"), use "sudo <command>".
See "man sudo_root" for details.

$ echo $SNAP_LIBRARY_PATH
/var/lib/snapd/lib/gl:/var/lib/snapd/lib/gl32:/var/lib/snapd/void
```

Though it's not strictly necessary for `jq` in the install hook it's in the generated command wrappers that snapcraft generates

Reply:
done